distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/CatalogBundleFacade.java

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package ddf.catalog.test;
+
+import static org.junit.Assert.fail;
+
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+
+import org.osgi.framework.InvalidSyntaxException;
+import org.osgi.framework.ServiceReference;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.CatalogFramework;
+import ddf.catalog.source.CatalogProvider;
+import ddf.catalog.source.FederatedSource;
+import ddf.common.test.ServiceManagerFacade;
+
+public class CatalogBundleFacade {
+    protected static final Logger LOGGER = LoggerFactory.getLogger(CatalogBundleFacade.class);
+
+    private final ServiceManagerFacade serviceManagerFacade;
+
+    public CatalogBundleFacade(ServiceManagerFacade serviceManagerFacade) {
+        this.serviceManagerFacade = serviceManagerFacade;
+    }
+
+    public CatalogProvider waitForCatalogProvider() throws InterruptedException {
+        LOGGER.info("Waiting for CatalogProvider to become available.");
+
+        CatalogProvider provider = null;
+        long timeoutLimit = System.currentTimeMillis() + TimeUnit.MINUTES.toMillis(5);
```

Comment:
We could still constant the TimeUnit version. But they're really only needed if more than one place is using them. I would almost say its even nearly pointless to make them constants. Are we going to keep making constants for every single different timeout? If we aren't going to do it for all and it doesn't shorten it much then I would say we just use TimeUnit where we need to.